README.md

```diff
@@ -241,6 +241,27 @@ Path to the shared file. Defaults to "#{Dir.home}/.aws/credentials".
 
 Defaults to 'default' or `[ENV]('AWS_PROFILE')`.
 
+### process_credentials
+
+This loads AWS access credentials from an external process.
+
+    <match *>
+      @type kinesis_streams
+
+      <process_credentials>
+        process CMD
+      </process_credentials>
+    </match>
+
+See also:
+
+*   [Aws::ProcessCredentials](https://docs.aws.amazon.com/sdk-for-ruby/v3/api/Aws/ProcessCredentials.html)
+*   [Sourcing Credentials From External Processes](https://docs.aws.amazon.com/cli/latest/topic/config-vars.html#sourcing-credentials-from-external-processes)
+
+**process**
```

Comment:
Agree with @riywo. I think `process` param should be `required`.

Reply:
Same as `role_arn` param in `assume_role_credentials`.
https://github.com/awslabs/aws-fluent-plugin-kinesis/pull/178/files/0e82e7fe74ca7241f25bad9ff5e843a173827bc5#diff-04c6e90faac2675aa89e2176d2eec7d8R152

Reply:
Woops, yes that should be required. I'll update this.